README.md

```diff
@@ -0,0 +1,45 @@
+# Eat Out Log
+
+### Project Status:
+
+##### Build information by branch
+
+master: [![Build Status](https://travis-ci.org/CaruskiApps/EatOutLog.svg?branch=master)](https://travis-ci.org/CaruskiApps/EatOutLog)
```

Comment:
These build status links will show the build (and eventually deploy) status of the 3 main branches.